### ISSUE TITLE
crisper favicon icons?

### ISSUE THREAD

**Issue by missinglink** [pelias/design#9]
Hi @elioqoshi 

These favicons looks a little blurry to my eye (compared to the vector), is there some way we could crisp them up a bit?

<img width="155" alt="Screenshot 2020-05-11 at 13 35 04" src="https://user-images.githubusercontent.com/738069/81557560-796b4880-938c-11ea-88f6-9f5e92feb51c.png">

**Comment by elioqoshi** [comment]
Hmm that's weird. It might be some anti-aliasing setting in the PNG export. Are you sure these are using the same resolution? Can you send the PNG's you used separately here?

cc @ergish

**Comment by missinglink** [comment]
It's literally just a screenshot of https://github.com/pelias/design#favicons you can click on them to see which is which.

This is the big one: https://github.com/pelias/design/blob/master/logo/pelias/Favicon/Favicon-PNG/Favicon_64x64.png

**Comment by missinglink** [comment]
Is it maybe because I'm on a retina display?

**Comment by missinglink** [comment]
Agh yeah that's it, I dragged the window to my second monitor and took another screenshot:

![Screenshot 2020-05-12 at 14 17 44](https://user-images.githubusercontent.com/738069/81690293-5f4c6b80-945b-11ea-8b1e-91eec1dd3f41.png)

[edit] ooor... not? I'm confused

**Comment by ergish** [comment]
Hi @missinglink 

I just checked the files again to be sure. The exported PNG files are PNG-24, so there's nothing left to do to make them look sharper. The second image looks ok so it may be caused by the different scaling and resolutions of the displays. 

I would personally recommend using SVG files in this case. They will look sharp in any display, resolution, or "zoom in" situation.

**Comment by elioqoshi** [comment]
> I would personally recommend using SVG files in this case. They will look sharp in any display, resolution, or "zoom in" situation.

I don't think SVG is possible for favicons usually.

But yeah, I think it's the retina display @missinglink, I get the same issue on my Mac.

**Comment by ergish** [comment]
Yeah, I also wasn't sure about it but it seems they are supported.
